test/e2e/WebpackProductionBuild.spec.ts

```diff
@@ -129,7 +129,7 @@ describe('Webpack Production Build', () => {
             '    4 |   const response = await fetch(',
           ].join('\n'),
           [
-            'ERROR in src/index.ts 2:29-43',
+            'ERROR in src/index.ts 2:29',
```

Comment:
Should this read `src/index.ts:2:29` instead? i.e. with a colon

Reply:
Yes, I missed out `IssueWebpackError.ts`, thanks